diversity-lunch-fe/src/components/VoucherPanel/VoucherPanel.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import '../../styles/component-styles/voucherpanel/voucherPanel.scss';
+import { useSelector } from 'react-redux';
+import { DiversityIconContainer } from '../General/HeaderTemplate/DiversityIconContainer';
+import { Button } from '../General/Button/Button';
+import { AppStoreState } from '../../store/Store';
+import { authenticatedFetchPut } from '../../utils/fetch.utils';
+import { AccountState, AccountStateOk } from '../../data/account/account-state.type';
+import { Account } from '../../types/Account';
+import { PopUp } from '../AdminPanel/userAdministration/PopUp';
+import { CloseSiteContainer } from '../General/HeaderTemplate/CloseSiteContainer';
+
+export const VoucherPanel = () => {
+    const [revealed, setRevealed] = useState(false);
+    const [voucherCode, setVoucherCode] = useState('empty');
+    const [isError, setError] = useState(false);
+    const accountState: AccountState = useSelector((store: AppStoreState) => store.account);
+    const account: Account = (accountState as AccountStateOk).accountData;
+
+    const extractMeetingIdFromURL = () => {
+        const url = String(window.location.pathname);
```

Comment:
Bitte url params benutzen: https://v5.reactrouter.com/web/example/url-params

Reply:
Die Doku oben ist nicht passend 
Hier für Typescript:
https://dev.to/javila35/react-router-hook-useparam-now-w-typescript-m93